samples/Transactions.ocf.json

```diff
@@ -443,35 +443,30 @@
       "security_id": "387878ba-8fb6-4673-812e-32c092947899",
       "date": "2017-07-22",
       "settlement_date": "2017-07-22",
+      "quantity": "9000",
       "release_price": {
         "amount": "9.00",
         "currency": "CAD"
-      }
+      },
+      "resulting_security_ids": []
     },
     {
       "object_type": "TX_PLAN_SECURITY_RELEASE",
       "id": "test-plan-security-release-full-fields",
       "security_id": "387878ba-8fb6-4673-812e-32c092947899",
       "date": "2017-07-22",
-      "settlement_date": "2017-07-22",
+      "settlement_date": "2017-07-25",
```

Comment:
When I was looking at this diff, I found myself confused by `date` vs `settlement_date`, so I looked at the schema and `settlement_date` is defined as "Date of settlement". Can we improve the documentation on this field?